Add album Living Things of artist Linkin Park

mikelyou opened the pull request:
My first time opening a pull request, please tell me if there's anything wrong, thanks.

snshn commented:
>   7: for no Windows-style EOLs within texts          FAILED (database-tests.at:46)

It also seems like you're using Windows, and line endings there are different from GNU/Linux or macOS -- two bytes instead of one. There should be a setting somewhere in your editor to switch to UNIX line endings (\n).

mikelyou commented:
Thanks for the helpful tips. I searched a bit for corresponding knowledge, and finally find out what's wrong and how to fix them. It seems I have fixed them since it passes the CI tests.

I'd like to summarize a bit about the problems, in case someone encounters this too in the future. Please correct me if I'm wrong.

### The problems
1. The files lack the newline at their end, appears as a red minus sign
![image](https://user-images.githubusercontent.com/58005137/87953323-e2e89100-ca78-11ea-92cc-5897b2316d70.png)
2. The line ending should be UNIX style `\n`(or LF), rather than Windows style `\r\n`(or CRLF). [A reference](https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types)

### Cause of problems
I guess I did a silly thing that I used **Microsoft Notepad** to edit the lyrics. This cause problem 2, and maybe also the other one, I don't know.

### Solve the problems
This is simple. I open those files in **Atom** and changed this place from `CRLF` to `LF`, and saved.
![image](https://user-images.githubusercontent.com/58005137/87955081-3eb41980-ca7b-11ea-86a4-26b9863b1651.png)

snshn commented:
[There it is](https://lyrics.github.io/db/L/Linkin%20Park/Living%20Things/)

Thanks Ü
